geode-core/src/main/java/org/apache/geode/management/internal/cli/CommandManager.java

```diff
@@ -14,293 +14,96 @@
  */
 package org.apache.geode.management.internal.cli;
 
-import static org.apache.geode.distributed.ConfigurationProperties.USER_COMMAND_PACKAGES;
 
-import java.lang.reflect.Method;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Properties;
-import java.util.ServiceConfigurationError;
-import java.util.ServiceLoader;
+import static java.util.stream.Collectors.toSet;
+
+import java.util.Objects;
 import java.util.Set;
-import java.util.StringTokenizer;
+import java.util.function.Predicate;
 
+import com.google.common.collect.ImmutableSet;
```

Comment:
It seems this is the only reason we're pulling in guava? I'd prefer not to pull in a new dependency if we don't need to (especially when it is not being used pervasively). It reduces the chance of future conflicts with user's application code which also uses that dependency.

Reply:
+1

Reply:
This is easy enough to remove.